modules/api/src/main/java/org/apache/ignite/sql/ResultSetImpl.java

```diff
@@ -15,25 +15,21 @@
  * limitations under the License.
  */
 
-package org.apache.ignite.internal.sql;
+package org.apache.ignite.sql;
```

Comment:
The class should be moved outside of API module, but you just move it to another package in the same module

Reply:
I've changed class visibility to package-private. 
Otherwise, we would have few copies of the same class in project. However, the class is just a sync adapter for async result set and provides no additional logic.